### PR TITLE
Add documentation for managing existing canasta instance

### LINKED ISSUE #18: Document the possibility to register an existing Canasta instance with the CLI
Just add the `myExistingCanastaInstance` object to `/etc/canasta/conf.json`'s `Installation` object:
```
{
       "Installations": {
               "myExistingCanastaInstance": {
                       "Id": "myExistingCanastaInstance",
                       "Path": "/home/canasta/myExistingCanastaInstance",
                       "Orchestrator": "docker-compose"
               }
       }
}
```

### PR DESCRIPTION
**Pull request by amalpaul54111**
Fixes #18

### CODE REVIEW

**Comment by yaronkoren** [comment]
This looks good to me, aside from a few small typos... anyone else?

File: README.md [hunk 1]
```diff
@@ -123,6 +123,25 @@ sudo canasta skin enable CologneBlue,Modern -i canastaId
...
+  "Installations": {
+    "wiki2": {
+      "Id": "myExistingCanastaInstance",
+      "Path": "/home/user/canasta/myExistingCanastaInstance",
+      "Orchestrator": "docker-compose"
+    }
+  }
+}
+```
+"Id"   - Represent the Canasta Id used by the cli to differentiate between installations.
```
Comment (jeffw16) [inline README.md]: ID* not "Id" (that's a Freudian concept)

File: README.md [hunk 2]
```diff
@@ -123,6 +123,25 @@ sudo canasta skin enable CologneBlue,Modern -i canastaId
...
+
+"Path" - The location where the configuration files(i.e. 'docker-compose.yaml' and other such files) are saved.
```
Comment (jeffw16) [inline README.md]: Please add a space between files and (i.e.

**Review by jeffw16 — CHANGES_REQUESTED** [review]
Please fix linguistic syntactical issues

**Comment by jeffw16** [comment]
Typos need to be resolved please.

**Comment by yaronkoren** [comment]
If we're already covering typos, I see a few more: "add a object" should be "add an object", and "Represent the Canasta ID used by the cli" should by "Represents the Canasta ID used by the CLI".

**Review by yaronkoren — APPROVED** [review]

**Comment by amalpaul54111** [comment]
Since, we are moving the CLI documentation to [Canasta-Documentation](https://github.com/CanastaWiki/Canasta-Documentation), this PR could be closed right? 
All the changes requested here have been implemented at https://github.com/CanastaWiki/Canasta-Documentation/pull/2

**Comment by yaronkoren** [comment]
Sure, let's just close it...

**Pull request closed without merging**